### PR TITLE
Add styling to site

### PR DESCRIPTION
**Pull request by ghost**
### What we did

* Style recipes in cards on (index, my profile , my collections'
* Style Show and New views
* Add image tag to recipe card
* Add styling to card
* Style input forms that effects (login, sing up)

image

### CODE REVIEW

**Comment by tochman** [comment]
What is the status on this one? Will this be done soon?

File: app/views/recipes/index.html.haml [hunk 1]
```diff
@@ -1,15 +1,16 @@
...
-      %p
-        Posted by:
-        = mail_to recipe.user.email
-      %br
+  %h3.text-center= category.name
+  .category.flex.flex-wrap.justify-center
+    - category.recipes.each do |recipe|
+      .recipe.bg-white.p-4.m-2.rounded.overflow-hidden.shadow-md.min-w-2xs.max-w-xs{id: "recipe-#{recipe.id}"}
+        = link_to (image_tag rails_blob_path(recipe.image), class: 'w-full'), recipe_path(recipe) unless recipe.image.attachment.nil?
+        -# = image_tag recipe.image, class: 'w-full' unless recipe.image.attachment.nil?
```
Comment (tochman) [inline app/views/recipes/index.html.haml]: Comments. Please remove

**Review by GeorgeJLewis — APPROVED** [review]
Looks good but update the PR description.

**Comment by tochman** [comment]
Yes, please do update the description

**Pull request merged**